tasks/java.yml

```diff
@@ -2,6 +2,8 @@
 - name: "Check if Java is installed"
   shell: which java
   register: check_java
+  failed_when: check_java.rc not in  (0, 1)
+  changed_when: False
```

Comment:
sorry i should say: still 1 (so err if java not installed). i.e. behavior is as before using the preferred `command -v`. so that change seems good to have.

the addition here of:

```yaml
failed_when: check_java.rc not in  (0, 1)
```

was just so the task is not considered failed when java is not installed (`.rc` 1). so no need to run w/ `--ignore-errors`.